src/styles/product-card.scss

```diff
@@ -0,0 +1,80 @@
+.product-card {
+  border: 1px solid #fff;
+  width: 200px;
+  background-color: #fff;
+  padding: 32px 16px 16px;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__name {
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    margin-bottom: 4px;
+    color: $main-text-color;
+  }
+
+  &__code {
+    color: #616070;
```

Comment:
The color value should be a variable instead of a hardcoded value to maintain consistency and ease of maintenance. Consider using a variable like $secondary-text-color.